iceberg/iceberg-handler/pom.xml

```diff
@@ -111,6 +111,10 @@
       <artifactId>RoaringBitmap</artifactId>
       <version>0.9.22</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+    </dependency>
```

Comment:
Why is this required? where is iceberg handler using this?

Reply:
It was in the list of commits for 1.3.0. But yes, it is unused in iceberg handler. Removed.

Reply:
Fixed.